Source/WebCore/page/scrolling/ScrollingStateScrollingNode.cpp

```diff
@@ -285,6 +285,11 @@ void ScrollingStateScrollingNode::setMouseIsOverContentArea(bool flag)
     setPropertyChanged(Property::ContentAreaHoverState);
 }
 
+void ScrollingStateScrollingNode::setMouseIsActiveInContentArea()
+{
+    setPropertyChanged(Property::MouseActivtyState);
```

Comment:
It’s weird to have a state bit but with no associated state. Previously we’ve had bugs when the scrolling state tree has changes that aren’t stateful (older versions of setRequestedScrollData were like this). But this seems OK for now.